tests/sports_test.py

```diff
@@ -27,7 +26,7 @@ def test_get_next_mens_basketball_game(self):
         self.assertIsInstance(sports.get_next_mens_basketball_game(), dict)
 
     def test_get_mens_basketball_standings(self):
```

Comment:
These unit tests where we just assert the type of return data type are weak. Can we instead match against a mock schema instead?

Reply:
Resolved in [3258f4b](https://github.com/pittcsc/PittAPI/pull/137/commits/3258f4b0e34c25565c99abf67ab9f9716c6fad5c)